bluebutton-data-pipeline-rif-extract/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/extract/s3/S3Utilities.java

```diff
@@ -29,13 +28,12 @@ public static AmazonS3 createS3Client(ExtractionOptions options) {
 
 	/**
 	 * @param awsS3Region
-	 *            the AWS {@link Region} that should be used when interacting
+	 *            the AWS {@link Regions} that should be used when interacting
 	 *            with S3
 	 * @return the {@link AmazonS3} client to use
 	 */
-	public static AmazonS3 createS3Client(Region awsS3Region) {
-		AmazonS3Client s3Client = new AmazonS3Client(new DefaultAWSCredentialsProviderChain());
-		s3Client.setRegion(awsS3Region);
+	public static AmazonS3 createS3Client(Regions awsS3Region) {
+		AmazonS3 s3Client = AmazonS3ClientBuilder.standard().withRegion(awsS3Region).build();
```

Comment:
This is the real change: `new AmazonS3Client(...)` is deprecated in the latest SDK version.

(The `setRegion(...)` call was actually already deprecated in the version we were using before.)